src/solvers/dgmulti/dg.jl

```diff
@@ -426,14 +504,39 @@ function calc_single_boundary_flux!(cache, t, boundary_condition, boundary_key,
 end
 
 
-# Todo: DGMulti. Specialize for modal DG on curved meshes using WADG
 # inverts Jacobian and scales by -1.0
 function invert_jacobian!(du, mesh::DGMultiMesh, equations, dg::DGMulti, cache; scaling=-1)
   @threaded for i in each_dof_global(mesh, dg, cache)
     du[i] *= scaling * cache.invJ[i]
   end
 end
 
+# inverts Jacobian using weight-adjusted DG, and scales by -1.0.
+# - Chan, Jesse, Russell J. Hewett, and Timothy Warburton.
+#   "Weight-adjusted discontinuous Galerkin methods: curvilinear meshes."
+#   https://doi.org/10.1137/16M1089198
```

Comment:
Could you quickly remind me what's needed to make this provably stable, please? I'm basically asking whether it would make sense to make the other DG solvers energy-stable with weak form volume terms, too.

Reply:
To make the entire formulation energy stable, we would also need a split form treatment of geometric terms in the volume integral. 

Weight-adjusted DG is basically just energy stable mass lumping, so you probably don't need it for `DGSEM` or `FDSBP` type solvers...